src/store/fileUploadStore.js

```diff
@@ -239,13 +241,11 @@ const actions = {
 			}
 
 			// Get localurl for some image previews
-			let localUrl = ''
+			let localUrl
 			if (SHARED_ITEM.MEDIA_ALLOWED_PREVIEW.includes(file.type)) {
 				localUrl = URL.createObjectURL(file)
 			} else if (isVoiceMessage) {
-				localUrl = file.localUrl
-			} else {
-				localUrl = OC.MimeType.getIconUrl(file.type)
+				localUrl = file.localURL
```

Comment:
```suggestion
				localUrl = file.localUrl
```
?
![image](https://github.com/user-attachments/assets/54c6dba4-c7e5-4ce6-8c45-3e0d1bea8159)

Reply:
Nope, that's `localURL` from File object, not localUrl from what we add to messageStore

Reply:
You are not wrong

Reply:
Time to align? :speak_no_evil: